### PR TITLE
[ES] Update Rules, Add COC and Abuse

### PR DESCRIPTION
**Pull request by Panchosauriooo**

### CODE REVIEW

File: wiki/Contributor_Code_of_Conduct/es.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+### 4. Prohibición permanente
+
+**Impacto en la comunidad**: Demostrar un patrón de violación de los estándares de la comunidad, incluyendo un constante comportamiento inapropiado, acoso de un individuo, o agresión o menosprecio de clases de individuos. 
+
+**Consecuencia**: Una prohibición permanente de cualquier tipo de interacción publica en la comunidad.
+
+## Atribución
+
+Este Código de Conducta es adapto de [Contributor Covenant][homepage], disponible en [https://www.contributor-covenant.org/version/2/0/code_of_conduct.html][v2.0].
```
Comment (MagoSimon) [inline wiki/Contributor_Code_of_Conduct/es.md:80]: Este Código de Conducta se adapto de [Contributor Covenant][homepage], disponible en [https://www.contributor-covenant.org/version/2/0/code_of_conduct.html][v2.0].

File: wiki/Rules/es.md [hunk 2]
```diff
@@ -12,8 +12,9 @@ Estas reglas son la base sobre la cual mantenemos la comunidad de osu! como un l
...
 3. **Se bueno con los demás.** El acoso o cualquier otro tipo de antagonismo no tiene lugar dentro de la comunidad de osu!. Estamos aquí para clickear círculos, golpear tambores, atrapar frutas y teclear hasta dieciocho teclas al mismo tiempo, no para ser idiotas los unos con los otros.
 4. **No seas un tipo desagradable.** Si en cualquier momento estás inseguro de si estás o no rompiendo esta regla, probablemente ya lo estás haciendo.
-5. **Estamos en una comunidad para todas las edades.** Esto significa que todo contenido para mayores de 18 años (18+) o inapropiado para espacios públicos (NSFW) como uso de drogas o tópicos de contenido sexual no son bienvenidos aquí. Se hacen excepciones en el caso de que sean apropiadas, pero en general, mantenlo decente.
-6. **Cuándo las reglas no prevalezcan, el sentido común lo hará.** La administración tiene discreción explícita de aplicar su juicio en caso de que sea necesario.
+5. **Estamos en una comunidad para todas las edades.** Esto significa que todo contenido para mayores de 18 años (18+) o inapropiado para espacios públicos (NSFW) como uso de drogas o tópicos de contenido sexual no son bienvenidos aquí.
```
Comment (TPGPL) [inline wiki/Rules/es.md:15]:
```suggestion
5. **Estamos en una comunidad para todas las edades.** Esto significa que todo contenido para mayores de 18 años (18+) o inapropiado para espacios públicos ([NSFW](https://es.wikipedia.org/wiki/NSFW)) como uso de drogas o tópicos de contenido sexual no son bienvenidos aquí.
```

File: wiki/Rules/es.md [hunk 3]
```diff
@@ -12,8 +12,9 @@ Estas reglas son la base sobre la cual mantenemos la comunidad de osu! como un l
 2. **Juega limpio.** Usar herramientas de terceros de cualquier tipo para obtener cualquier forma de ventaja no está bien. Esto incluye cosas como macros, programas de asistencia de puntería, modificación de escalas de tiempo, y demases. Si algún programa te está ayudando a hacer algo que deberías hacer tú mismo en el juego, no está bien.
```
Comment (TPGPL) [inline wiki/Rules/es.md:12]: - Check if the article is fully up-to-date with the English version
   - If yes, delete the `outdated` mark at the beginning
- Link at line 22 to `Reporting Bad Behaviour` is missing
- `Reglas de Envio de Beatmaps` header is outdated
- Link at line 68 is missing (the whole header might be outdated?)

File: wiki/Contributor_Code_of_Conduct/es.md [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+
+Las Directrices del Impacto en la Comunidad fueron inspiradas por [Mozilla's code of conduct enforcement ladder][Mozilla CoC].
```
Comment (TPGPL) [inline wiki/Contributor_Code_of_Conduct/es.md:82]: "Mozilla's code of conduct enforcement ladder" should be possible to translate

File: wiki/Reporting_Bad_Behaviour/Abuse/es.md [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+
+Si el tweet proviene de una cuenta que esta enlazada al perfil de osu! a través de nuestra página (aparecerá debajo de su avatar y fecha de unión al lado de un icono con estilo de un pájaro), por favor envía un correo a [abuse@ppy.sh](mailto:abuse@ppy.sh) conteniendo una copia del tweet en forma de imagen donde sea posible y cualquier otra información en la que puedas pensar.
+
+## Un miembro de uno de los equipos voluntarios (NAT, GMT) está acercándose/hablándome/actuando de manera inapropiada hacia mí!
+
+Envía un correo a [abuse@ppy.sh](mailto:abuse@ppy.sh) conteniendo la mayor cantidad posible de información **inmediatamente**.
+
+Si cualquier cosa que incluyas te hace sentir avergonzado, por favor déjalo en blanco. Esto incluye capturas de pantalla, registros del chat, básicamente cualquier cosa que puedas adjuntar a un correo.
+
+Lo investigaremos tan pronto como podamos.
```
Comment (TPGPL) [inline wiki/Reporting_Bad_Behaviour/Abuse/es.md:70]: one header is missing

```md
## Someone I know is being abused by a member of the volunteer teams (NAT, GMT) but is too afraid to speak up!

Send an email to [abuse@ppy.sh](mailto:abuse@ppy.sh) containing everything you know with as much evidence as you can get.

If you don't have any evidence but only stories or rumors that you believe are substantial enough to act on, please send us an email anyway. We take complaints like this very seriously and will look deeply into every report.```

**Review by TPGPL — CHANGES_REQUESTED** [review]

File: wiki/Reporting_Bad_Behaviour/Abuse/es.md [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+Después, contacta a los administradores del servidor de inmediato, buscando en la parte superior de la lista de usuarios e informales de que ha pasado. Si no puedes averiguar quién es un administrador, busca a alguien que tenga una corona al lado de su nombre – él es el dueño del servidor y es responsable de lo que ocurra dentro de él.
+
+Si el abuso continúa y los administradores no hacen nada al respecto, [por favor notifica al equipo de Confianza y seguridad de Discord](https://dis.gd/request).
+
+Adicionalmente, si conoces el nombre de usuario de osu! de la persona que está molestándote, por favor envía un correo a [abuse@ppy.sh](mailto:abuse@ppy.sh) conteniendo la mayor cantidad posible de información y este será investigado prontamente.
+
+### En un Twitter enlazado a un perfil de osu!
+
+Haz click en los tres puntos en la parte superior del tweet conteniendo el problema y haz click en 'Reportar Tweet'. Asegúrate de seguir los formularios y de leerlos cuidadosamente. Quizás también quieras leer [La guía de Twitter acerca de cómo reportar comportamiento abusivo](https://help.twitter.com/en/safety-and-security/report-abusive-behavior) para más información acerca del proceso.
```
Comment (TPGPL) [inline wiki/Reporting_Bad_Behaviour/Abuse/es.md:56]:
```suggestion
Haz click en los tres puntos en la parte superior del tweet conteniendo el problema y haz click en 'Reportar Tweet'. Asegúrate de seguir los formularios y de leerlos cuidadosamente. Quizás también quieras leer [La guía de Twitter acerca de cómo reportar comportamiento abusivo](https://help.twitter.com/es/safety-and-security/report-abusive-behavior) para más información acerca del proceso.
```

File: wiki/Reporting_Bad_Behaviour/Abuse/es.md [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+
+Después, puedes ya sea bloquear o silenciar al usuario en cuestión haciendo click en los tres puntos en la parte superior derecha del su tweet (si estas usando la página web o el cliente móvil). Consulta [La guía de Twitter acerca de cómo bloquear cuentas](https://help.twitter.com/en/using-twitter/blocking-and-unblocking-accounts) para más información.
```
Comment (TPGPL) [inline wiki/Reporting_Bad_Behaviour/Abuse/es.md:58]:
```suggestion
Después, puedes ya sea bloquear o silenciar al usuario en cuestión haciendo click en los tres puntos en la parte superior derecha del su tweet (si estas usando la página web o el cliente móvil). Consulta [La guía de Twitter acerca de cómo bloquear cuentas](https://help.twitter.com/es/using-twitter/blocking-and-unblocking-accounts) para más información.
```

File: wiki/Reporting_Bad_Behaviour/Abuse/es.md [hunk 8]
```diff
@@ -0,0 +1,77 @@
...
+Envía un correo a [abuse@ppy.sh](mailto:abuse@ppy.sh) conteniendo la mayor cantidad posible de información **inmediatamente**.
+
+Si cualquier cosa que incluyas te hace sentir avergonzado, por favor déjalo en blanco. Esto incluye capturas de pantalla, registros del chat, básicamente cualquier cosa que puedas adjuntar a un correo.
+
+Lo investigaremos tan pronto como podamos.
+
+## Alguien que conozco está siendo abusado por un miembro de los equipos voluntarios (NAT, GMT) pero esta muy asustado para hablar al respecto!
+
+Envía un correo a [abuse@ppy.sh](mailto:abuse@ppy.sh) conteniendo todo lo que sabes con la mayor cantidad de evidencia que puedas obtener.
+Send an email to [abuse@ppy.sh](mailto:abuse@ppy.sh) containing everything you know with as much evidence as you can get.
```
Comment (TPGPL) [inline wiki/Reporting_Bad_Behaviour/Abuse/es.md:75]:
```suggestion
```

**Review by TPGPL — APPROVED** [review]

**Pull request merged**